code/data_structures/src/maxSubArray(KadaneAlgorithm)/KadaneAlgorithm.cpp

```diff
@@ -0,0 +1,30 @@
+#include <bits/stdc++.h>
```

Comment:
it's a bad practice to use such a header, it can be useful only in comptetive programming

Reply:
I understand, I have changed the header files into the ones required.